openbb_terminal/stocks/government/quiverquant_view.py

```diff
@@ -135,8 +135,7 @@ def display_government_buys(
 
     ax.set_ylabel("Amount [1k $]")
     ax.set_title(
-        f"Top {limit} purchased stocks over last {past_transactions_months} "
-        f"months (upper bound) for {gov_type.upper()}"
+        f"{gov_type.upper()}'s top {limit} purchased stocks in last {past_transactions_months} months"
```

Comment:
Thanks for this @reshuraj57 !! 🙏🏽

Can we add the word `(upper)` in the title? Just because the data we get is a range, and in this we output the upper value.

Reply:
@DidierRLopes I have updated the title as per your request.

Reply:
@DidierRLopes can u review and approve PR please?

Reply:
Absolutely @reshuraj57 !

Sorry, I missed your message! Thank you for working on this 😄